cookbooks/aws-parallelcluster-slurm/files/default/head_node_checks/check_cluster_ready.py

```diff
@@ -112,13 +112,21 @@ def check_deployed_config_version(cluster_name: str, table_name: str, expected_c
 
         missing, incomplete, wrong = _check_cluster_config_items(instance_ids, items, expected_config_version)
 
-        if missing or incomplete or wrong:
+        if incomplete or wrong:
```

Comment:
*Late Comment*
[Test] I think we should cover it with integ test. It is enough to modify the existing test covering the update and submit a job that spawns a dynamic compute node while the update is ongoing.